hacktoberfest-signin/src/assets/data/users.json

```diff
@@ -1,17 +1,32 @@
 [
-    {
-        "heroimage": "https://avatars0.githubusercontent.com/u/36649901?v=4",
-        "username": "GoodGuyGregory",
-        "languages": {
-            "python": 4,
-            "java": 3,
-            "c++": 3,
-            "f#": 2,
-            "ruby": 4,
-            "swift": 1,
-            "go": 3,
-            "javascript": 4,
-            "c#": 3
-        }
+  {
+    "heroimage": "https://avatars0.githubusercontent.com/u/36649901?v=4",
+    "username": "GoodGuyGregory",
+    "languages": {
+      "python": 4,
+      "java": 3,
+      "c++": 3,
+      "f#": 2,
+      "ruby": 4,
+      "swift": 1,
+      "go": 3,
+      "javascript": 4,
+      "c#": 3
     }
+  },
+  {
```

Comment:
This is the element I will need for your data to be added. sorry if the instructions were unclear.

Reply:
just your user object inside of the assets data.json